users/tests/test_users.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from users.models import Role, User
+from django.contrib.auth.models import User as DjangoUser
+from falcon.test_support import Random
+from django.core.exceptions import ValidationError
+
+
+@pytest.mark.django_db
+class TestUsers:
+
+    def test_create_user(self, employee_1):
+        assert isinstance(employee_1, User)
+        assert User.objects.filter(user=employee_1.user).exists()
+
+    def test_delete_user(self, employee_1):
+        userId = employee_1.user.id
+        employee_1.delete()
+        assert not User.objects.filter(user=userId).exists()
+        assert not DjangoUser.objects.filter(pk=userId).exists()
+
+    def test_create_user_without_email(self, team_1):
```

Comment:
maybe you can do part of the tests with parameterize to avoid repetition of the same code

Reply:
I am not agree, its not readable and not consistent with the rest of our project.
I think that creating objects in the contest and use them is much clear and easy to understand.

Reply:
Duplication of code is problematic:
1. because it is hard for maintenance
2. it can lead to inconsistency when you change one place and forget to change on other occurrences.

when you have to change one thing if all in the same function/test it's is easy and safer to change it.
also, it is easy to add one more test in the same concept like an invalid field for each field of the object.

you can see example parametrize code in:
https://github.com/redhat-beyond/falcon/blob/main/tasks/tests/test_create_task.py#L68

Reply:
Its not readable and if you want to maintain your data close to the tests we can create the fixture in the class itself

Reply:
Further more, in [here](https://github.com/redhat-beyond/falcon/blob/main/tasks/tests/test_create_task.py#L68)  you need to the same amount of data which is the same as creating fixture. But with the parameterize you don't give the field's names which is not readable at all and pretty much the same amount of code.
A better solution was to create FACTORY which I tried to use in the start but you rejected it.

Reply:
I agree with Aviv here.
Anyway, I think it is out of the scope of this PR, the point here was just to refactor some of the already existing tests and change them to use fixtures from conftest.py.